android/app/src/main/java/app/organicmaps/util/SharingUtils.java

```diff
@@ -20,6 +29,8 @@ public class SharingUtils
   private static final String KMZ_MIME_TYPE = "application/vnd.google-earth.kmz";
   private static final String TEXT_MIME_TYPE = "text/plain";
 
+  private static Uri sourceFileUri;
```

Comment:
Is using static variable to pass URL the best way? @rtsisyk WDYT?

Reply:
I agree, using a global variable is not beautiful solution. But, unfortunately, Intent with the Intent.ACTION_CREATE_DOCUMENT type does not support passing custom parameters out of the box. And making some kind of wrapper is quite difficult and dangerous from the point of view of bugs.